Jenkinsfile

```diff
@@ -1,5 +1,113 @@
+// Copyright 2018 SUSE LINUX GmbH, Nuernberg, Germany.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 def targetBranch = env.getEnvironment().get('CHANGE_TARGET', env.BRANCH_NAME)
 
 library "kubic-jenkins-library@${targetBranch}"
 
-coreKubicProjectCi()
+// TODO: Move this whole check into a function
+if (env.CHANGE_AUTHOR != null) {
+    // TODO: Don't hardcode salt repo name, find the right place
+    // to lookup this information dynamically.
+    githubCollaboratorCheck(
+        org: 'kubic-project',
+        repo: 'salt',
+        user: env.CHANGE_AUTHOR,
+        credentialsId: 'github-token')
+}
+
+// Configure the job properties
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '15')),
+    disableConcurrentBuilds(),
+])
+
+withKubicEnvironment() {
+    stage('Setup Helm Client') {
+        helmInstallClient()
+    }
+
+    stage('Services Tests') {
+        // TODO: Hardcoding this list of charts and values isn't nice...
+        parallel 'Helm: MariaDB': {
+            // TDOO: Helm: MariaDB is ran by the "core" CI too. Move to
+            // a shared function.
+            String releaseName = "helm-" + UUID.randomUUID()
+
+            helmInstallChart(
+                environment: environment,
+                releaseName: releaseName,
+                chartName: "stable/mariadb",
+                wait: true,
+                values: [
+                    service: [
+                        type: "NodePort"
+                    ],
+                    persistence: [
+                        enabled: false
+                    ]
+                ]
+            )
+
+            // TODO: Test if MariaDB is up and reachable and...
```

Comment:
The upstream mariadb chart includes one test we can try, though it is guaranteed to fail if a pod is not ready yet (referring to your --wait comment) 
https://github.com/kubernetes/charts/blob/master/stable/mariadb/templates/test-runner.yaml
 
`helm test $releaseName`

Reply:
Aha - I didn't know Helm included this. This looks perfect, though it seems we'll need to wait for helm 1.8.1+ to get the `--wait` fix.

Reply:
bump - looks like 1.8.2 just arrived